internal/core/tests.rs

```diff
@@ -61,32 +61,45 @@ pub extern "C" fn slint_send_mouse_click(
 #[no_mangle]
 pub extern "C" fn send_keyboard_string_sequence(
     sequence: &crate::SharedString,
-    modifiers: KeyboardModifiers,
     window_adapter: &crate::window::WindowAdapterRc,
 ) {
     for ch in sequence.chars() {
-        let mut modifiers = modifiers;
         if ch.is_ascii_uppercase() {
-            modifiers.shift = true;
+            WindowInner::from_pub(window_adapter.window()).process_key_input(KeyInputEvent {
+                event_type: KeyEventType::KeyPressed,
+                text: Key::Shift.into(),
+                ..Default::default()
+            });
         }
-        let mut buffer = [0; 6];
-        let text = SharedString::from(ch.encode_utf8(&mut buffer) as &str);
+        let text = SharedString::from(ch);
 
-        WindowInner::from_pub(window_adapter.window()).process_key_input(&KeyEvent {
+        WindowInner::from_pub(window_adapter.window()).process_key_input(KeyInputEvent {
             event_type: KeyEventType::KeyPressed,
             text: text.clone(),
-            modifiers,
             ..Default::default()
         });
-        WindowInner::from_pub(window_adapter.window()).process_key_input(&KeyEvent {
+        WindowInner::from_pub(window_adapter.window()).process_key_input(KeyInputEvent {
             event_type: KeyEventType::KeyReleased,
             text,
-            modifiers,
             ..Default::default()
         });
     }
 }
 
+/// Simulate a character input event.
+#[no_mangle]
+pub extern "C" fn send_keyboard_char(
```

Comment:
no_mangle  functions should start with `slint_`   why do we need this anyway?

Reply:
I refactored that a little bit. At the moment you can only send a string sequence (press and imminently release). But I need to send a press key e.g. for a shift modifier and send released on a specific line.

Reply:
yes, looks like an oversight.  we don't want to polite the global namespace of symbol.